x/blob/types/payforblob_test.go

```diff
@@ -239,47 +240,52 @@ func TestNewMsgPayForBlob(t *testing.T) {
 	tests := []test{
 		{
 			signer:      addr.String(),
-			nid:         []byte{1, 2, 3, 4, 5, 6, 7, 8},
-			blob:        []byte{1},
+			nids:        [][]byte{{1, 2, 3, 4, 5, 6, 7, 8}},
+			blobs:       [][]byte{{1}},
+			versions:    make([]uint8, 1),
```

Comment:
[blocking][discussion] I would expect this test case to fail because it uses an unsupported share version. Do you think `CreateMultiShareCommitment` should perform validation on the share versions provided?

Reply:
> it uses an unsupported share version

wait, I'm confused, why is this? isn't it zero? perhaps we should be more explicit and use the const.

> should perform validation on the share versions provided?

imo no, because we don't want to add state to an otherwise stateless function. All verification of share version should occur during the creation process of a PFB

Reply:
> wait, I'm confused, why is this? isn't it zero? perhaps we should be more explicit and use the const.

Sorry I misread. On initial read I thought this was attempting to use the share version 1. You're right it is zero and I agree the const would be clearer.

> All verification of share version should occur during the creation process of a PFB

Fair enough